addons/binding/org.openhab.binding.groheondus/README.md

```diff
@@ -0,0 +1,46 @@
+# GROHE ONDUS Binding
+
+The GROHE ONDUS Binding provides access to data collected by a GROHE ONDUS appliance, such as an (GROHE Sense Guard)[https://www.grohe.de/de_de/smarthome/grohe-sense-guard/].
+The binding uses the REST API interface (the same as used by the Android App) to retrieve the collected data.
+
+## Supported Things
+
+This binding should support all appliances from GROHE, however, only the GROHE Sense Guard is tested with it.
+
+## Discovery
+
+The binding requires you to create at least one Account interface thing as a bridge manually. After you did that (e.g. using the PaperUI or configuring the thing as described in the example) you can start a discover of items (the discovering will not start itself).
```

Comment:
* "Account interface" Maybe just call this thing an Account?
* PaperUI -> Paper UI
* Put each sentence on its own line here.
* Discovery services discover things and not items
* Normally discovery is triggered periodically. Did you disable this for some reason?

Reply:
> Normally discovery is triggered periodically. Did you disable this for some reason?
You're right, it's possible that I just manually started the discovery process so I did not need to wait for it to start automatically :P

Otherwise: All other points are done! :)